publisher-sdk/src/androidTest/java/com/criteo/publisher/SessionTest.kt

```diff
@@ -21,9 +21,11 @@ import com.nhaarman.mockitokotlin2.spy
 import com.nhaarman.mockitokotlin2.whenever
 import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
+import kotlin.time.ExperimentalTime
 
 class SessionTest {
 
+  @ExperimentalTime
```

Comment:
What is this used for ?

Reply:
oops left over from when I was playing with `Duration` experimental APIs. Removed.